Cargo.toml

```diff
@@ -28,9 +28,9 @@ overflow-checks = true
 #backtraces = ["cosmwasm-std/backtraces"]
 
 [dependencies]
-cosmwasm-std = { version = "1.2.3" }
-cosmwasm-storage = { version = "1.2.3", features = ["iterator"] }
-cw-storage-plus = { version = "1.0.1" }
+cosmwasm-std = { version = "1.3.3" }
+cw-storage-plus = { version = "1.1.0" }
+cosmwasm-storage = { version = "1.3.3", features = ["iterator"] }
```

Comment:
This will be removed in a later PR in this series  (https://github.com/FigureTechnologies/ats-smart-contract/pull/121/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542L33)